### PR TITLE
Added 358-suspend-mechanism-switching-S0ix-S3.md

### PR DESCRIPTION
**Pull request by johanes2115**

### CODE REVIEW

File: docs/unified-test-documentation/dasharo-compatibility/358-suspend-mechanism-switching-S0ix-S3.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
+# Dasharo Compatibility: Suspend mechanism switching (S0ix/S3)
```
Comment (sulewskiprzemyslaw) [inline docs/unified-test-documentation/dasharo-compatibility/358-suspend-mechanism-switching-S0ix-S3.md:1]: @johanes2115 please add relevant entry point to `mkdocs` file. Without it, the test suite will remain hidden

File: docs/unified-test-documentation/dasharo-compatibility/358-suspend-mechanism-switching-S0ix-S3.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+
+The output from the command above should contain the phrase `deep` enclosed
+in square brackets, indicating that the Suspend to RAM (S3) mode is
+enabled.
+
+Example output:
+
+```bash
+s2idle [deep]
+```
```
Comment (sulewskiprzemyslaw) [inline docs/unified-test-documentation/dasharo-compatibility/358-suspend-mechanism-switching-S0ix-S3.md:110]: @johanes2115 as this option is currently enabled for the NC laptops, please add this test suite to test-matrix for laptops

**Review by sulewskiprzemyslaw — CHANGES_REQUESTED** [review]

File: docs/unified-test-documentation/dasharo-compatibility/358-suspend-mechanism-switching-S0ix-S3.md (hunk 1 above)
Comment (johanes2115) [reply docs/unified-test-documentation/dasharo-compatibility/358-suspend-mechanism-switching-S0ix-S3.md:1]: @sulewskiprzemyslaw I added the entrypoint to mkdocs.md file.

File: docs/unified-test-documentation/dasharo-compatibility/358-suspend-mechanism-switching-S0ix-S3.md (hunk 2 above)
Comment (johanes2115) [reply docs/unified-test-documentation/dasharo-compatibility/358-suspend-mechanism-switching-S0ix-S3.md:110]: @sulewskiprzemyslaw I updated the test matrix files.

**Review by sulewskiprzemyslaw — APPROVED** [review]

**Pull request merged**